### PR TITLE
Enable unit test lit testing with default vitis sysroot 

### PR DESCRIPTION
**Pull request by denolf**

### CODE REVIEW

File: runtime_lib/libxaie/CMakeLists.txt [hunk 1]
```diff
@@ -42,4 +42,4 @@ include_directories(
 
 add_library(xaiengine SHARED ${libsources})
 
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libxaiengine.so DESTINATION ${CMAKE_INSTALL_PREFIX}/runtime_lib)
\ No newline at end of file
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libxaiengine.so DESTINATION ${CMAKE_INSTALL_PREFIX}/runtime_lib/libxaie)
```
Comment (jgmelber) [inline runtime_lib/libxaie/CMakeLists.txt:45]: This should match the name of the library: xaiengine or aie-rt if we choose to move to that repo

File: test/CMakeLists.txt [hunk 2]
```diff
@@ -53,14 +53,20 @@ find_package(Vitis)
...
+  set(LibXAIE_ROOT ${VitisSysroot}/usr ${VitisSysroot}/opt/xaiengine ${VitisSysroot}/opt/xaienginev2)
+  find_package(LibXAIE)
+  if(LibXAIE_FOUND)
+    message(STATUS "Using LibXAIE in sysroot: ${XILINX_XAIE_DIR}")
+  else()
+    message(STATUS "LibXAIE NOT FOUND in ${VitisSysroot}: Skipping some tests")
+  endif()
 else()
-message(STATUS "LibXAIE NOT FOUND: Skipping some tests")
+  set(LibXAIE_ROOT ${CMAKE_INSTALL_PREFIX}/runtime_lib/libxaie)
```
Comment (jgmelber) [inline test/CMakeLists.txt:65]: Match library name

File: tutorials/CMakeLists.txt [hunk 3]
```diff
@@ -53,14 +53,20 @@ find_package(Vitis)
...
+  set(LibXAIE_ROOT ${VitisSysroot}/usr ${VitisSysroot}/opt/xaiengine ${VitisSysroot}/opt/xaienginev2)
+  find_package(LibXAIE)
+  if(LibXAIE_FOUND)
+    message(STATUS "Using LibXAIE in sysroot: ${XILINX_XAIE_DIR}")
+  else()
+    message(STATUS "LibXAIE NOT FOUND in ${VitisSysroot}: Skipping some tests")
+  endif()
 else()
-message(STATUS "LibXAIE NOT FOUND: Skipping some tests")
+  set(LibXAIE_ROOT ${CMAKE_INSTALL_PREFIX}/runtime_lib/libxaie)
```
Comment (jgmelber) [inline tutorials/CMakeLists.txt:65]: Match library name

File: tutorials/makefile-common [hunk 4]
```diff
@@ -25,4 +25,4 @@ CHESS_FLAGS = -P ${VITIS_AIE_INCLUDE_DIR}
 # The following additional flags are only applied for host code. We add the
 # necessary search paths for the sysroot so clang++ can find the aarch64
 # includes and libraries.
-AIECC_HOST_FLAGS += -I$(AIE_INSTALL)/runtime_lib $(AIE_INSTALL)/runtime_lib/test_library.cpp -I${VITIS_AIETOOLS_DIR}/include/drivers/aiengine -I${MLIR_AIE_SYSROOT}/usr/include -I${MLIR_AIE_SYSROOT}/usr/include/c++/${LIBCXX_VERSION} -I${MLIR_AIE_SYSROOT}/usr/include/c++/${LIBCXX_VERSION}/aarch64-xilinx-linux -L${MLIR_AIE_SYSROOT}/usr/lib/aarch64-xilinx-linux/${LIBCXX_VERSION} -L/$(AIE_INSTALL)/runtime_lib -B${MLIR_AIE_SYSROOT}/usr/lib/aarch64-xilinx-linux/${LIBCXX_VERSION}
+AIECC_HOST_FLAGS += -I$(AIE_INSTALL)/runtime_lib $(AIE_INSTALL)/runtime_lib/test_library.cpp -I${VITIS_AIETOOLS_DIR}/include/drivers/aiengine -I${MLIR_AIE_SYSROOT}/usr/include -I${MLIR_AIE_SYSROOT}/usr/include/c++/${LIBCXX_VERSION} -I${MLIR_AIE_SYSROOT}/usr/include/c++/${LIBCXX_VERSION}/aarch64-xilinx-linux -L${MLIR_AIE_SYSROOT}/usr/lib/aarch64-xilinx-linux/${LIBCXX_VERSION} -L/$(AIE_INSTALL)/runtime_lib/libxaie -B${MLIR_AIE_SYSROOT}/usr/lib/aarch64-xilinx-linux/${LIBCXX_VERSION}
```
Comment (jgmelber) [inline tutorials/makefile-common:28]: Match library name

**Review by jgmelber — CHANGES_REQUESTED** [review]
I think the installation directory should match the library name in the external repo to avoid confusion. It is worth considering that we build the [aie-rt](https://github.com/Xilinx/aie-rt) repo for this update as well.

**Comment by fifield** [comment]
One thing to note here is that the Makefiles in the tutorial will run against whatever is in your `PATH` whereas the lit configuration knows where to find the binaries in the build tree. This might require extra setup in CI including creating an install before running the tutorial tests. The makefiles will also make it more difficult to put build artifacts in the correct place; I'm guessing that's why the tests need a `make clean` line.

File: runtime_lib/libxaie/CMakeLists.txt (hunk 1 above)
Comment (denolf) [reply runtime_lib/libxaie/CMakeLists.txt]: Done. matched to xaiengine until aie-rt gets updated.

File: test/CMakeLists.txt (hunk 2 above)
Comment (denolf) [reply test/CMakeLists.txt]: Done. matched to xaiengine until aie-rt gets updated.

File: tutorials/CMakeLists.txt (hunk 3 above)
Comment (denolf) [reply tutorials/CMakeLists.txt]: Done. matched to xaiengine until aie-rt gets updated.

**Review by jgmelber — APPROVED** [review]

**Review by jackl-xilinx — APPROVED** [review]

**Comment by denolf** [comment]
Merged #379

**Pull request merged**